src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -312,6 +261,94 @@ private void ProcessObjectContent(PSObject inputObject)
                     WriteHexidecimal(inputBytes, null, 0);
                 }
             }
+            else
+            {
+                string errorMessage = StringUtil.Format(UtilityCommonStrings.FormatHexTypeNotSupported, obj.GetType());
+                ErrorRecord errorRecord = new ErrorRecord(new ArgumentException(errorMessage),
+                                                            "FormatHexTypeNotSupported",
+                                                            ErrorCategory.InvalidArgument,
+                                                            obj.GetType());
+                WriteError(errorRecord);
+            }
+        }
+
+        private byte[] ConvertToByteArray(object inputObject)
+        {
+            if (inputObject is string str)
+            {
+                return Encoding.GetBytes(str);
+            }
+
+            var baseType = inputObject.GetType();
+            byte[] result = null;
+            int elements = 1;
+            bool isArray = false;
+            bool isBool = false;
+            bool isEnum = false;
+            if (baseType.IsArray)
+            {
+                baseType = baseType.GetElementType();
+                dynamic o = inputObject;
```

Comment:
Is this used? I don't see this used anywhere after this code point.

Reply:
In next line. Name is very short and not readable.